websites/A/AnimeHeaven/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "compilerOptions": {
+    "outDir": "./dist/"
+  },
+  "extends": "../../../tsconfig.json"
+}
```

Comment:
Use from docs...
```
{
  "extends": "../../../tsconfig.json",
  "compilerOptions": {
    "outDir": "./dist/"
  }
}
```

Reply:
It's the same thing, just different order in the file?

Reply:
Yeah 🤷